layouts/shortcodes/box.html

```diff
@@ -0,0 +1,17 @@
+<!--Notice Color Scheme Fix-->
+<!--Credits: https://github.com/martignoni/hugo-notice-->
```

Comment:
The license is GPLv3, this might be problematic to use?

Reply:
Hmm... 
> Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Shouldn't be a problem for such a small modification... Moreover, the repo has been properly credited as a submodule.

Reply:
The problem is that we would have to make all the code of the website GPL to comply with the license terms. I don't know if that is a problem or not.

Reply:
Can't we just mark the file as GPLv3 instead (like just mentioning in it)... If not, then I could just remove the file for the time being.

Reply:
The mainroad theme is actually GPL2, so probably OK? I'll think more about this and come back to you.

Reply:
Well we are already using a hugo-notice submodule so whatever.